library/plugins/rethinkdb/rethinkdb-publisher/src/test/kotlin/com/cdsap/talaiot/publisher/rethinkdb/RethinkDbPublisherTest.kt

```diff
@@ -1,10 +1,8 @@
-package com.cdsap.talaiot.publisher
+package com.cdsap.talaiot.publisher.rethinkdb
 
 import com.cdsap.talaiot.configuration.RethinkDbPublisherConfiguration
```

Comment:
We should remove this import since it points to an invalid path (`RethingDbPublisherConfiguration` resides in `com.cdsap.talaiot.publisher.rethinkdb`)
```suggestion
```

Reply:
thx, done